docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -133,6 +133,8 @@ quarkus.oidc.client-id=quarkus-app
 
 # This is a key which will be used to retrieve a secret from the map of credentials returned from CredentialsProvider
 quarkus.oidc.credentials.client-secret.provider.key=mysecret-key
+# this is the keyring provided to the CredentialsProvider when looking up the secret
```

Comment:
Please tune the message to explain when setting this property is required but let users know when they can omit it